src/View/Characters/Characters.test.jsx

```diff
@@ -0,0 +1,31 @@
+import { screen, render } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import Characters from './Characters';
+import { character } from '../../mocks/TestData';
+import { MemoryRouter } from 'react-router-dom';
+
+const server = setupServer(
+  rest.get('https://api.disneyapi.dev/characters', (req, res, ctx) => {
+    return res(ctx.json(character));
+  })
+);
+
+beforeAll(() => server.listen());
+afterAll(() => server.close());
+
+test('should render header and character names with images', async () => {
+  render(
+    <MemoryRouter initialEntries={['/characters']} initialIndex={0}>
```

Comment:
You shouldn't need the initialEntires here. Just wrapped on MemoryRouter since this looks like a fetch all.